src/formatDate.js

```diff
@@ -50,7 +50,54 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
+  const separatorTo = toFormat.pop();
+  const oldDateFormat = date.split(separatorFrom);
+  const newDateFormat = [];
+  const LONG_YEAR_FORMAT = 'YYYY';
+  const SHORT_YEAR_FORMAT = 'YY';
+  const MONTH_FORMAT = 'MM';
+  const DAY_FORMAT = 'DD';
+  const CENTURY_20 = '19';
+  const CENTURY_21 = '20';
+
+  let oldYearIndex = fromFormat.indexOf(LONG_YEAR_FORMAT);
+
+  if (oldYearIndex === -1) {
+    oldYearIndex = fromFormat.indexOf(SHORT_YEAR_FORMAT);
+  }
+
+  let newYearIndex = toFormat.indexOf(LONG_YEAR_FORMAT);
+
+  if (newYearIndex === -1) {
+    newYearIndex = toFormat.indexOf(SHORT_YEAR_FORMAT);
+  }
+
+  let year = oldDateFormat[oldYearIndex];
+  const SHORTENED_YEAR = year.slice(-2);
```

Comment:
It should not be in `UPPER_CASE`
constant value -> `UPPER_CASE`
calculated value -> `camelCase`